frontend/src/features/backend/backend.tsx

```diff
@@ -137,7 +137,11 @@ export function BackendConfig(props: BackendConfigProps) {
 
   return (
     <>
-      <Offcanvas show={props.show} onHide={props.handleClose}>
+      <Offcanvas
+        className={lato.className}
```

Comment:
this is pretty gross but is necessary for the font to work in `Offcanvas` in the compiled static site due to how next.js handles fonts